ow_lander/scripts/arm_move_joints_client.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+# The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+# Research and Simulation can be found in README.md in the root directory of
+# this repository.
+
+import rospy
+import actionlib
+from actionlib_msgs.msg import GoalStatus
+import ow_lander.msg
+import argparse
+from guarded_move_action_client import print_arguments
+from urdf_parser_py.urdf import URDF
+from distutils.util import strtobool
+
+
+def ArmMoveJoints_client():
+
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('relative', type=strtobool,
+                        help='Move joint relative to the current position', nargs='?', default='False', const=0)
+
+    parser.add_argument('angles', type=float,
+                        help='goal angle of the joints in radians', nargs=6, default=[0.1, 0.1, 0.1, 0.1 ,0.1, 0.1])
+    args = parser.parse_args()
+    print_arguments(args)
+
+    client = actionlib.SimpleActionClient(
+        'ArmMoveJoints', ow_lander.msg.ArmMoveJointsAction)
+
+    client.wait_for_server()
+
+    goal = ow_lander.msg.ArmMoveJointsGoal()
+
+    goal.relative = bool(args.relative)
+    #goal.joint = args.joint
+    goal.angles = args.angles
```

Comment:
This combination of an optional boolean argument and required array of floats is always required when put in this order. I have not been able to produce a situation where either of them are truly optional. I think it would make sense to instead expand `angles` out into the various named joint angles in order of their index. 
```suggestion
    parser.add_argument('relative', type=strtobool,
                        help='Move joint relative to the current position', nargs='?', default='False', const=0)

    parser.add_argument('j_shou_yaw', type=float,
        help='Shoulder yaw joint angle', nargs='?', default=0.1, const=0)
    parser.add_argument('j_shou_pitch', type=float,
        help='Shoulder pitch joint angle', nargs='?', default=0.1, const=0)
    parser.add_argument('j_prox_pitch', type=float,
        help='Proximal pitch joint angle', nargs='?', default=0.1, const=0)
    parser.add_argument('j_dist_pitch', type=float,
        help='Distal pitch joint angle', nargs='?', default=0.1, const=0)
    parser.add_argument('j_hand_yaw', type=float,
        help='Hand yaw joint angle', nargs='?', default=0.1, const=0)
    parser.add_argument('j_scoop_yaw', type=float,
        help='Scoop yaw joint angle', nargs='?', default=0.1, const=0)
    args = parser.parse_args()
    print_arguments(args)

    client = actionlib.SimpleActionClient(
        'ArmMoveJoints', ow_lander.msg.ArmMoveJointsAction)

    client.wait_for_server()

    goal = ow_lander.msg.ArmMoveJointsGoal()

    goal.relative = bool(args.relative)
    goal.angles = [
        args.j_shou_yaw,
        args.j_shou_pitch,
        args.j_prox_pitch,
        args.j_dist_pitch,
        args.j_hand_yaw,
        args.j_scoop_yaw
    ]
```

Reply:
Done!

Reply:
as discussed we will have a separate jira to introduce the joint limits for all the joints and then we can revisit the problem of range of allowed angles for the joints.